docs/content/direct/knownissue-wsl-ghcr-helm.md

```diff
@@ -1,20 +1,62 @@
-# Authorization failure in WSL while fetching Helm chart from ghcr.io
+# Authorization failure in WSL/Ubuntu while fetching Helm chart from ghcr.io
 
 ## Description of the Issue
 
-When using WSL (Windows Subsystem for Linxu) and following the
-[Getting Started recipe](get-started.md) you might get a failure from
-the command to instantiate KubeStellar's core Helm chart. The error
-message is as follows.
+When working with WSL (Windows Subsystem for Linux)/Ubuntu and trying to instantiate KubeStellar's core Helm chart, you might encounter the following error:
```

Comment:
Because the issue is not specific to WSL nor Ubuntu, I would remove "working with WSL (Windows Subsystem for Linux)/Ubuntu and".